corehq/apps/app_manager/templates/app_manager/v2/partials/releases.html

```diff
@@ -12,9 +12,21 @@
        });
     });
     $(function () {
+        hqImport('hqwebapp/js/urllib.js').registerUrl(
+            'form_data',
+            '/a/' + '{{ domain }}' + '/apps/view/---/form_data/'
```

Comment:
huh i never knew about that. played with it a bit briefly and got some errors, i'll update in a followup. (have another PR for case diff stuff)